csrankings-g.csv

```diff
@@ -520,6 +520,7 @@ Gitta Domik,Paderborn University,https://cs.uni-paderborn.de/cgvb/personal/gitta
 Gitta Domik-Kienegger,Paderborn University,https://cs.uni-paderborn.de/cgvb/personal/gitta-domik-kienegger,HP9yrOkAAAAJ
 Gitta Kutyniok,LMU Munich,https://www.math.lmu.de/~kutyniok,JHs9LssAAAAJ
 Giulia Boato,University of Trento,https://webapps.unitn.it/du/en/Persona/PER0000755/Curriculum,_nFMOZ8AAAAJ
+Giulia Guidi,Cornell University,http://gguidi.com/,UZLC4TYAAAAJ

```

Comment:
Home page immediately redirects to https://giuliaguidi.github.io/.

Reply:
Is it a problem? Should I update the csv with https://giuliaguidi.github.io/?

Reply:
I prefer stable links, and redirects are often flaky. You can also just set http://gguidi.com to point directly (not as a redirect) [to your GitHub pages site](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site).